CODEOWNERS

```diff
@@ -1,3 +1,9 @@
 * @DataDog/dd-trace-js
 /packages/dd-trace/src/appsec/ @DataDog/appsec-js
 /packages/dd-trace/test/appsec/ @DataDog/appsec-js
+/packages/datadog-plugin-*/ @Datadog/apm-framework-integrations-reviewers-js
+/packages/datadog-instrumentations/ @Datadog/apm-framework-integrations-reviewers-js
+/packages/ddtrace/src/plugins/ @Datadog/apm-framework-integrations-reviewers-js
```

Comment:
this folder includes a lot of files, some of which are not exclusive to APM, so I wouldn't include it here

Reply:
The non-APM ones AFAICT are `ci_plugin` and `log_plugin`. I can leave these to dd-trace-js (you might want to grab ownership of ci_plugin in #3415 anyway)